astropy/tests/helper.py

```diff
@@ -192,11 +194,72 @@ def assert_quantity_allclose(actual, desired, rtol=1.0e-7, atol=None, **kwargs):
     This is a :class:`~astropy.units.Quantity`-aware version of
     :func:`numpy.testing.assert_allclose`.
     """
-    import numpy as np
-
     from astropy.units.quantity import _unquantify_allclose_arguments
 
     __tracebackhide__ = True
     np.testing.assert_allclose(
         *_unquantify_allclose_arguments(actual, desired, rtol, atol), **kwargs
     )
+
+
+def assert_objects_equal(actual, desired, /, *, attrs=None, compare_class=True):
+    """Compare objects by their shape and attributes, including those provided by .info
+
+    Parameters
+    ----------
+    actual and desired: np.ndarray (positional only)
+
+    attrs: list[str] (optional, keyword only)
+       a list of attributes to compare in addition to the default list:
```

Comment:
Clean up indentation? 4 spaces. Also below for `compare_class` and `AssertionError`

Reply:
done